.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,36 @@
-Describe the pull request here, including any supplemental information needed to understand it.
+resolves #
 
-### This pull request changes...
+**Description-**
+
+**This pull request changes...**
 
 - expected change 1
 - expected change 2
 
-### This is how to verify this change...
+**This pull request also touches…**
 
-### This pull request is ready to merge when...
+- possible side effects of this change
 
-- [ ] Develop has updated automated tests (and all tests are passing)
-- [ ] This code has been reviewed by someone other than the original author
-- [ ] QA has manually tested and created bugs for any issues found (beyond minor fixes)
-- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
+**This pull request was tested in the follow ways…**
```

Comment:
These were performed by the developer or the reviewer?  If this is the template I use to open a PR it seems like the developer, but also the final tester probably wants to add to this list.

Reply:
I was looking at this as the developer letting the QA know what was already tested. It was a suggestion made by Richie and how his team is currently working. We can try it out and see if we would like to keep it.